Library/brew.sh

```diff
@@ -70,6 +70,10 @@ then
     HOMEBREW_RUBY_PATH="/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby"
   else
     HOMEBREW_RUBY_PATH="$(which ruby)"
+    if [[ -z "$HOMEBREW_RUBY_PATH" ]]
```

Comment:
I was wondering about that but there will be readable failures below in that case. I was just trying to address the slightly confusing error when you try to `exec` an empty string.